app/views/options-on-start-page/check-state-pension-1.html

```diff
@@ -0,0 +1,136 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block proposition_header %}
+  <!-- blank to remove the proposition header -->
+{% endblock %}
+
+{% block header_class %}
+  <!-- blank to remove the proposition header -->
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+
+  <div class="breadcrumbs">
+    <ol>
+      <li><a href="#">Home</a></li>
+      <li><a href="#">Working, jobs and pensions</a></li>
+      <li><a href="#">State Pension</a></li>
+    </ol>
+  </div>
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <h1 class="heading-xlarge">
+        {% if serviceName %} {{ serviceName }} {% endif %}
+      </h1>
+
+      <p>Use this service to find out:</p>
+
+      <ul class="list list-bullet">
+        <li>how much State Pension you could get (this amount is also known as your State Pension forecast)</li>
+        <li>when you can get it</li>
+        <li>how to increase it, if you can</li>
+      </ul>
+
+      <div role="note" class="panel panel-border-wide">
+        <p>
+          This page is also available in Welsh (<a href="#">Cymraeg</a>).
+        </p>
+      </div>
+      <p>
+        <a href="https://www.gov.uk/government/news/proposed-new-timetable-for-state-pension-age-increases">The State Pension age is under review</a>
+        and may change in the future.
+      </p>
+      <div role="note" class="panel panel-border-wide">
+        <p>
+          You can’t use this service if you’re already getting your State Pension or if you’ve delayed (‘deferred’) claiming it.
+        </p>
+      </div>
+      <p>
+      </p>
+
+      <h2 class="heading-medium">Sign in with Gateway</h2>
+      <p>
+        Lorem ipsum dolor sit amet, nec cu mazim ignota nemore, no dicant iuvaret patrioque his, sea ei nibh audire inciderint. Wisi atqui mei id, ne quo sonet convenire. Tota philosophia ea pri, vivendo appareat mel ex, mea cu dicta everti. His ne commune noluisse reprehendunt, nam eu dicit sapientem referrentur. Vero option civibus vix et.
+      </p>
+      <div class="left">
+        <a class="button" href="https://www.tax.service.gov.uk/gg/sign-in?continue=%2Fcheck-your-state-pension%2Faccount&origin=nisp-frontend&accountType=individual" role="button">Sign in with Gateway</a>
```

Comment:
Should this be 'Government Gateway'?

Reply:
It does just say "Gateway" on the designs. We should probably update them too.